app/assets/stylesheets/application.scss

```diff
@@ -1,8 +1,164 @@
 @charset "utf-8";
-
 @import "normalize-rails";
-@import "bourbon";
-@import "base/grid-settings";
-@import "neat";
-@import "base/base";
-@import "refills/flashes";
+@import "bootstrap/bootstrap";
+// For the google Font 'arvo': font-family: 'Arvo', georgia, serif;
+// The red I'm using is hex #cd2027
+body, html {
+	height: 100%;
+	width: 100%;
+	background-image: -webkit-linear-gradient(#61CCA2 0%, #B29772 100%);
+}
+h1, h2, h3, h4, h5, h6 {
+	font-family: 'bebasregular', 'open sans', 'helvetica', sans-serif;
+	color: white;
+	text-transform: uppercase;
+}
+a, div a {
+	font-family: 'bebasregular', 'open sans', 'helvetica', sans-serif;
+}
+p {
+	font-family: 'Arvo', georgia, serif;
+	color: white;
+}
+.submark {
+	height: 50px;	
+}
+.navbar-default, .btn, .btn-default, button, div {
+	border-radius: 0px;
+} 
+// CSS for the landing page ...
+#main-logo {
+	height: 30em;
+}
+.logo {
+	max-width: 900px;
+	width: 66%	
+}
+#description {
+	font-size: 175%;
+	margin: 50px 1em 50px 1em;
+	padding: 2em 5em 2em 5em;
+}
+.signup, .signup a {
+	color: white;
+	background: #cd2027;
+	border: 2px solid #cd2027;
+	-webkit-transition: all 0.6s;
+	-moz-transition: all 0.6s;
+	transition: all 0.6s;
+}
+.signup:hover, .signup a:hover {
+	color: #cd2027;
+	background: white;
+	border: 2px solid #cd2027;
+}
+
+#signup-btn {
+	height: 200px;
+}
+.btn-signup, .btn-signup:visited {
```

Comment:
Rule set contains (11/10) properties<br>Each selector in a comma sequence should be on its own single line